presto-orc/src/main/java/com/facebook/presto/orc/OrcSelectiveRecordReader.java

```diff
@@ -679,6 +683,8 @@ public Page getNextPage()
             }
         }
 
+        systemMemoryUsage.newOrcLocalMemoryContext(OrcSelectiveRecordReader.class.getSimpleName()).setBytes(INSTANCE_SIZE + getSelfRetainedSizeInBytes());
+
```

Comment:
we add ```INSTANCE_SIZE``` here as well as on L740 - is that intentional?

Reply:
Thanks for the catch @sujay-jain